### DISCUSSION TITLE
some restrictions not working

### DISCUSSION THREAD

**Discussion by moonraver** [osmandapp/OsmAnd#18126] [Maps Related]
Hello
There is a problem with navigation in Osmand map.

1. "restriction=no_entry" is not working at all
2. "restriction=no_straight_on" is working but without "except=bicycle".

Thank you.

**Comment by DmitryAlexei** [comment]
- Restriction=no_entry 1180 objects according to [Taginfo](https://taginfo.openstreetmap.org/tags/restriction=no_entry#overview)
- Restriction=no_straight_on + except=bicycle    620 ways and 365 relations according to Overpass-turbo.eu

<img width="692" alt="Screenshot 2023-09-11 001632" src="https://github.com/osmandapp/OsmAnd/assets/116079541/0700e910-5364-4d1f-b14c-ab1a492d2382">

**Reply by moonraver** [reply]
I think you have misunderstood me.................these types of restrictions are not working in Osmand Android app  (Slovakia map). Actually I have the latest night build 4.6.0#741m.

**Comment by DmitryAlexei** [comment]
These tags are not used in our routing (see https://github.com/osmandapp/OsmAnd-resources/blob/master/routing/routing.xml). Perhaps because there are not many such objects in OSM.

**Comment by moonraver** [comment]
So could you add at least "restriction=no_entry" (with it's available exceptions) to your routing?
It should solve problems with navigation into "false oneway streets".
Thank you

**Comment by scaidermern** [comment]
>     1. "restriction=no_entry" is not working at all
>     2. "restriction=no_straight_on" is working but without "except=bicycle".

These are known issues, see #13049 and #3882.
